project_task_pull_request_state/models/project_task.py

```diff
@@ -0,0 +1,108 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, fields, models
+
+
+class ProjectTaskState(models.Model):
+    _inherit = "project.task"
+
+    pr_state = fields.Selection(
+        selection=lambda self: self._selection_pr_state(),
+        tracking=True,
+        copy=False,
+        string="PR State",
+        compute="_compute_pr_state",
+        readonly=False,
```

Comment:
I would suggest adding [precompute=True](https://github.com/odoo/odoo/blob/32d159921443daa3cf559bcc2cd605094e34b6ff/addons/crm/models/res_partner.py#L14). In this case computed values will be updated when dependency is triggered in the frontend. So user will be aware of changes and can put a custom value if required.

Reply:
applied